### ISSUE TITLE
Please improve the documentation

### ISSUE THREAD

**Issue by FSpanhel** [usernamehw/vscode-run-commands-view#1]
I have absolutely no clue what this extension is about...

**Comment by usernamehw** [comment]
@FSpanhel Yeah, I was delaying adding `.gif` because it's rather new extension and the size would change from `14Kb` to `500Kb` and the frequent updates redownload the entire extension each time.

Will add in the next update.

Just one question though: Is it not clear before or after installing?
